tracer/dependabot/Datadog.Dependabot.Integrations.csproj

```diff
@@ -158,8 +158,8 @@
 
     <!-- Integration: NUnit3.TestAdapter -->
     <!-- Assembly: NUnit3.TestAdapter -->
-    <!-- Latest package https://www.nuget.org/packages/NUnit3TestAdapter/4.0.0 -->
-    <PackageReference Include="NUnit3TestAdapter" Version="4.0.0" />
+    <!-- Latest package https://www.nuget.org/packages/NUnit3TestAdapter/4.1.0 -->
+    <PackageReference Include="NUnit3TestAdapter" Version="4.1.0" />
```

Comment:
I don't think this file is working correctly - it's listing the latest package versions we _could_ support, not the latest we _do_ support 🤔 We don't have support for NUnit v4 yet for example, but this file suggests we do.